arches_for_science/utils/digital_resource_for_manifest.py

```diff
@@ -0,0 +1,217 @@
+                service_identifier_tile.data[service_identifier_type_node_id] = service_identifier_type
+                service_identifier_tile.data[service_identifier_content_node_id] = build_string_object(service_identifier_content)
+                service_identifier_tile.save(transaction_id=transactionid, index=False)
+
+    if id:
+        for identifier_content, identifier_type in id.items():
+            identifier_tile = Tile.get_blank_tile_from_nodegroup_id(nodegroup_id=identifier_nodegroupid, resourceid=resource_id)
+            identifier_tile.data[identifier_content_node_id] = build_string_object(identifier_content)
+            identifier_tile.data[identifier_type_node_id] = identifier_type
+            identifier_tile.save(transaction_id=transactionid, index=False)
+
+    if type:
+        type_tile = Tile.get_blank_tile_from_nodegroup_id(nodegroup_id=type_nodegroupid, resourceid=resource_id)
+        type_tile.data[type_node_id] = type
+        type_tile.save(transaction_id=transactionid, index=True)
+
+def create_digital_resource(instance, iiif_type, canvas=None):
+    """
+        Creates the digital resources resource instance representing manifest
+        and also creates the manifest_x_canvas record
+    """
+    iiif_manifest_value = ["305c62f0-7e3d-4d52-a210-b451491e6100"]
+    internal_id_valueid = ["768b2f11-26e4-4ada-a699-7a8d3fe9fe5a"]
+    web_service_valueid = ['e208df66-9e61-498b-8071-3024aa7bed30']
+    digital_resource_graph = "707cbd78-ca7a-11e9-990b-a4d18cec433a"
+
+    manifest_data = instance.manifest
+    service = {manifest_data['@context']: web_service_valueid} #TODO canvas does not have its own service
+    type=iiif_manifest_value #TODO canvas type need to be added
+    transactionid=instance.transactionid
+
+    resource = Resource(graph_id = digital_resource_graph)
+    resource.save(transaction_id=instance.transactionid)
+    resource_id = resource.pk
+
+    if iiif_type == "manifest":
+        name = manifest_data["label"]
+        print("manifest_name",name)
+        statement = manifest_data["description"]
+        id = {str(instance.globalid): internal_id_valueid}
+        service_identifiers = [
+            {manifest_data["@id"]: ["f32d0944-4229-4792-a33c-aadc2b181dc7"]},
+        ]
+    elif iiif_type == "canvas":
+        name = canvas["label"]
+        print("canvas name",name)
+        statement = None
+        id = {canvas["images"][0]["resource"]["service"]["@id"]: internal_id_valueid}
+        service_identifiers = [
+            {canvas["images"][0]["resource"]["@id"]: ["f32d0944-4229-4792-a33c-aadc2b181dc7"]},
+            {canvas["images"][0]["resource"]["service"]["@id"]: ["768b2f11-26e4-4ada-a699-7a8d3fe9fe5a"]},
+            {canvas["images"][0]["@id"]: ["768b2f11-26e4-4ada-a699-7a8d3fe9fe5a"]},
+        ]
+    print("type",type)
+    print("name",name)
```

Comment:
remove print statement - also on lines 138 and 146

Reply:
thanks